src/rust/lib/graph-editor/src/component/edge.rs

```diff
@@ -328,28 +530,63 @@ pub fn sort_hack_2(scene:&Scene) {
 // === FRP ===
 // ===========
 
+/// FRP endpoints for aggregated mouse events.
+#[derive(Clone,CloneRef,Debug)]
+#[allow(missing_docs)]
+pub struct ShapeViewEventsProxy {
```

Comment:
Looking at the docs and the name of this struct I do not understand what it is used for.